pkg/app/ops/orphancommandcleaner/orphancommandcleaner.go

```diff
@@ -0,0 +1,71 @@
+package orphancommandcleaner
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/pipe-cd/pipe/pkg/datastore"
+	"github.com/pipe-cd/pipe/pkg/model"
+)
+
+var (
+	commandTimeOut = 7 * 24 * time.Hour
+	interval       = 24 * time.Hour
```

Comment:
What values should be set?

Reply:
`commandTimeOut = 24 * time.Hour`
` interval = 6 * time.Hour`